translations/game_modes.tex/pl.po

```diff
@@ -137,6 +130,9 @@ msgid ""
 "A 1 vs.~1 mode with different way of building Map and scoring. See the Tournament Book.\n"
 "\n"
 msgstr ""
+"\\subsection*{Turniej}\n"
+"Tryb 1 na 1 z innym sposobem budowania Mapy oraz punktacją. Opisany w Księdze Turniejów.\n"
```

Comment:
z rulebooka PL (SG) ten proces to "przygotowywanie mapy", ew. inny cytat "ułożenia kafelków Mapy."